intellij-toml/src/main/kotlin/org/toml/lang/TomlFileTypeDetector.kt

```diff
@@ -12,8 +12,12 @@ import com.intellij.openapi.vfs.VirtualFile
 import org.toml.lang.psi.TomlFileType
 
 class TomlFileTypeDetector : FileTypeRegistry.FileTypeDetector {
-    override fun getVersion(): Int = 1
+    override fun getVersion() = 1
 
     override fun detect(file: VirtualFile, firstBytes: ByteSequence, firstCharsIfText: CharSequence?): FileType? =
         if (file.name == "config" && file.parent?.name == ".cargo") TomlFileType else null
+
+    override fun getDetectedFileTypes() = listOf(TomlFileType)
+
+    override fun getDesiredContentPrefixLength() = 0
```

Comment:
Specify return type explicitly here and above, please

Reply:
Done. I am curious what is rationale behind such decision? https://kotlinlang.org/docs/reference/coding-conventions.html has examples with the opposite.

Reply:
I personally prefer explicit returning types for functions because it simplifies readability and prevents unexpected errors when you change the expression type and as a result function returning type is also changed.
Of course, in case of overridden method, the last argument not relevant so it's just for consistency here

Also, Rust uses the same rules :)

> https://kotlinlang.org/docs/reference/coding-conventions.html

Interesting. But I didn't find explicit conventions about returning type - only examples for other things